backend/app/core/redis.py

```
@@ -0,0 +1,45 @@
+
+    @classmethod
+    async def get_client(cls) -> redis.Redis:
+        if cls._instance is None:
+            async with cls._lock:
+                if cls._instance is None:
+                    # Mask credentials for logging
+                    log_url = settings.redis_url
+                    if "@" in log_url:
+                        try:
+                            schema, rest = log_url.split("://", 1)
+                            auth, host = rest.split("@", 1)
```

Ruff (0.14.14)

Unpacked variable `auth` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)